pick.lic

```diff
@@ -231,6 +232,11 @@ class Pick
     @equipment_manager.wear_items(@removed_items)
   end
 
+  def balance_lockpick_container
+    return unless @use_lockpick_ring and @balance_lockpick_container
+    bput("turn my #{@settings.lockpick_container} to best", "You fiddle with", "You think about it")
```

Comment:
This version of pick doesn't do the Ruby include of any commons to make it easier to unit test, so you will need to prefix this bput with DRC.